xpublish_opendap/dap_xarray.py

```diff
@@ -34,56 +39,66 @@ def dap_dtype(da: xr.DataArray):
         return dap.String
 
 
-def dap_attribute(key: str, value):
-    """Create a DAP attribute."""
+def dap_attribute(key: str, value: Any) -> dap.Attribute:
+    """Create a DAP attribute"""
     if isinstance(value, int):
         dtype = dap.Int32
     elif isinstance(value, float):
         dtype = dap.Float64
     else:
         dtype = dap.String
-    return dap.Attribute(name=key, value=value, dtype=dtype)
+
+    return dap.Attribute(
+        name=key,
+        value=value,
+        dtype=dtype,
+    )
```

Comment:
It looks like some of the other pre-commit failures are where autopep8 is [conflicting with black's formatting](https://github.com/xpublish-community/xpublish-opendap/actions/runs/4833578323/jobs/8613781508?pr=3#step:4:110).

Reply:
Fixed